README.md

```diff
@@ -7,3 +7,67 @@ We're using the [OpenWeather API](https://openweathermap.org/current) to get wea
 You'll need to get your own API Key from their website (for free) and add it as an environment variable in a `.env` file. We have a template available as `example.env`.
 
 You'll be using React initially to build this. If you're new to React, check out the [website](https://reactjs.org) for some information on getting started! 
+
+# Project Setup for Windows OS:
+### 1. Fork the project:
+> _This creates a copy of the project into repositories of your GitHub account_
+
+### 2. Clone the forked repository:
+
+  You need to clone (download) it to your local machine using the below command in terminal
+```bash
+   $ git clone https://github.com/Your_username/prep-project-23.MAR.PREP.2.git
+```
+> Replace `Your_username` with your github username because your cloning the forked repository that exists in your github repositories.
+> _This creates a local copy of the repository in your local machine_
+
+### 3. Navigate to the forked repository:
+Once you have cloned the `prep-project-23.MAR.PREP.2` repository into your local machine, move into that folder using the change directory `cd` command on Linux/ Mac/ Windows
+```bash
+   $ cd prep-project-23.MAR.PREP.2
+```
+run the below command if you wish to open the project in Visual Studio code editor
+```
+  $ code .
+```
+### 4. Installing node and npm:
+You need to install node into your machine. navigate to [Node.js Download Page](https://nodejs.org/en/download/) and download the latest version that suits your OS and system architecture(x32, x64, x86)
+check the version of node using below command:
+```bash
+   $ node -v
+```
+
+npm comes installed with node, check npm version using below command:
+```bash
+   $ npm -v
+```
+
+> you can install nvm(optional), if you want to manage your node versions which needing to uninstall and install everytime when you want to change the version of node.
+
+### 5. starting the development server:
+After node and npm are ready on your system, now run the below command to install the dependencies if any:
+```bash
+   $ npm install
+```
+
+If it says there are some vulnerabilities, try running below command to fix some of the non-critical ones:
+```bash
+   $ npm audit fix
+```
+
+Then run the below command to start the development server:
+```bash
+   $ npm start
+```
+
+if the react-scripts version is < 5.0.0 then it might throw this error `Error: error:0308010c:digital envelope routines::unsupported`, then try updating your react-scripts version. To do that run the following commands:
+```bash
+   $ npm uninstall react-scripts
+   $ npm install react-scripts
+```
```

Comment:
Explain that this is one reason for the issue - "Check your react-scripts version in package.json and if it is less than <5.0.0, upgrade it".

There are multiple other reasons for this error - and then people can investigate and also check the freecodecamp link.